### PR TITLE
Use this.options in subgenerator && fix Object.assign

### PR DESCRIPTION
**Pull request by zckrs**

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports = fountain.Base.extend({
       }
 
       this.prompt(prompts, props => {
-        Object.assign(this.props, this.options, props);
+        Object.assign(this.props, this.props, props);
```
Comment (zckrs) [inline generators/app/index.js]: @Swiip with this change we can directly call framework generator.

**Comment by micaelmbagira** [comment]
Need to be rebased

**Comment by zckrs** [comment]
Rebased

**Comment by micaelmbagira** [comment]
👍

File: generators/app/index.js (hunk 1 above)
Comment (Swiip) [inline generators/app/index.js]: Ok but it should be `Object.assign(this.props, props);` :)

**Comment by Swiip** [comment]
👍

File: generators/app/index.js (hunk 1 above)
Comment (zckrs) [inline generators/app/index.js]: :joy: 
Fixed

**Pull request merged**